Makefile

```diff
@@ -6,6 +6,10 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+bootstrap:
+	go get github.com/stretchr/testify/assert
```

Comment:
Is testify needed here for bootstrapping?  when running 'make test' it should get all the packages required by the taskcluster-worker